payment_term_type/README.md

```diff
@@ -0,0 +1,8 @@
+Payment Terms Type
+==================
+
+This module allow to show in payment terms the payment type depending of the lines to compute
+
+* Credit
+* Cash
```

Comment:
La manera correcta de decir "De contado" es: Cash Down.
La manera correcta de decir "A credito" es: Pay by Credit / Paying for by credit.

Saludos.